components/raftstore-v2/src/fsm/peer.rs

```diff
@@ -249,3 +262,33 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> PeerFsmDelegate<'a, EK, ER,
         self.fsm.peer.propose_pending_writes(self.store_ctx);
     }
 }
+
+impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> PeerFsmDelegate<'a, EK, ER, T> {
```

Comment:
Why put it in a separate `impl` block?

Reply:
I think the impl above relates to the general logic. Is it better to put them together?

Reply:
I don't have any preference. I was just a bit curious about it.

You can add a bit of comments to the impl block.